src/common/options.cc

```diff
@@ -2349,6 +2349,11 @@ std::vector<Option> get_global_options() {
     .set_default(10.0)
     .set_description("Seconds before in-flight op is considered 'laggy' and we query mon for the latest OSDMap"),
 
+    Option("objecter_homeless_timeout", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
+    .set_default(0)
+    .set_description("Timeout homeless session : Duration (in fractional seconds) after which the op is cancelled")
+    .set_long_description("It will cancel the op associated with a homeless session after the specified duration."),
```

Comment:
as far as i know, the term 'homeless' isn't used in any of our docs, so this needs a better description

Reply:
@cbodley I updated the long description